packs/libcloud/actions/create_vm.py

```diff
@@ -32,11 +31,10 @@ def run(self, credentials, name, size_id=None, image_id=None, size_name=None, im
             size = [s for s in driver.list_sizes() if size_name in s.name][0]
 
         if image_id is not None:
-            image = NodeImage(id=image_id, name=None,
-                              driver=driver)
+            image = [i for i in driver.list_images() if image_id == i.id][0]
```

Comment:
Also, now that I think of it, you probably got `AttributeError` / `KeyError` when trying to create a GCE node so I see how debugging this could waste a bunch of time :/